lib/mongoose_push/telemetry.ex

```diff
@@ -0,0 +1,31 @@
+defmodule MongoosePush.Telemetry do
+  @moduledoc """
+  Module responsible for handling telemetry events
+  """
+  alias MongoosePush.Telemetry.Handler
+
+  def handler do
+    attach_metrics()
```

Comment:
This feels weird. We attach handlers each time we call function that returns supervisor spec? This seems like side effect that is hard to expect. I would much prefer this to be just a separate function, like. `attach_all/0` that is meant to be explicitly called when application starts.